### ISSUE TITLE
gnome3.gnome-keyring fails test suite on i686

### ISSUE THREAD

**Issue by peti** [NixOS/nixpkgs#51121]
I recently tried to install NixOS with gnome on a i686 machine, but ran into trouble when `gnome-keyring` would not pass its test suite. The issue does not seem to exist on `x86-64`.  The issue exists in both recent versions of `nixos-18.09` as well as `master`:

~~~
ERROR:pkcs11/gkm/test-data-der.c:639:test_write_pkcs8_encrypted: assertion failed: (res == GKM_DATA_SUCCESS)
FAIL: test-data-der 18 /gkm/data-der/write_pkcs8_encrypted
ERROR: test-data-der process failed: 250

ERROR:pkcs11/gnome2-store/test-gnome2-private-key.c:147:test_save_private_key: assertion failed: (gkm_data_der_read_private_pkcs8_crypted (data, "booo", 4, &sexp) == GKM_DATA_SUCCESS)
FAIL: test-gnome2-private-key 2 /gnome2-store/private-key/save
ERROR: test-gnome2-private-key process failed: 250
~~~

**Comment by jtojnar** [comment]
The test suite sometimes fails on x86_64 as well: https://github.com/NixOS/nixpkgs/pull/45950#issuecomment-438295424

**Comment by abathur** [comment]
I'm taking the fact that #55293 was closed by 884f7ad3cf87455b283ba0a47a9e873bb84ced7b as justification for closing this one as well (especially since comments late in that thread note that other distros disabled or didn't count these tests). Happy to reopen if anyone thinks I'm overreaching :)
